ts/components/wallet/TransactionsList.tsx

```diff
@@ -165,11 +169,15 @@ export default class TransactionsList extends React.Component<Props> {
 
         <Grid>
           <Row>
-            <List
+            <FlatList
               scrollEnabled={false}
               removeClippedSubviews={false}
-              dataArray={transactions as Transaction[]} // tslint:disable-line: readonly-array
-              renderRow={this.renderRow}
+              data={transactions as Transaction[]} // tslint:disable-line: readonly-array
```

Comment:
you could remove cast and `// tslint:disable-line: readonly-array` too

Reply:
updated within faf2362d2d0cdb1546a638bb4f9e8a1d5624c7e3